packages/share_plus/share_plus/pubspec.yaml

```diff
@@ -39,7 +39,7 @@ dependencies:
   url_launcher_linux: ^3.0.5
   url_launcher_platform_interface: ^2.1.2
   ffi: ^2.0.1
-  web: ^0.5.0
+  web: ">=0.5.0 <2.0.0"
```

Comment:
Should be set to `^1.0.0` and the code updated accordingly.

Reply:
Updated, considering that I removed the extension a few days ago.